.slather.yml

```diff
@@ -12,3 +12,4 @@ ignore:
   - "Sources/StreamChatUI/StreamSwiftyGif"
   - "StreamChatUITestsApp"
   - "StreamChatUITestsAppUITests"
+  - "**/*/Deprecations.swift"
```

Comment:
This should not be part of this PR, right? Adam is already fixing it in his PR

Reply:
This is related to Sonar, not SwiftLint. I'm not aware of Adam working on Sonar stuff 🤔

Reply:
Yeah, discard my comment!